### PR TITLE
Gives curators a console for printing portraits not in the round onto canvases

### PR DESCRIPTION
**Pull request by tralezab**
## About The Pull Request

To be more accurate, it adds a program that requires curator access to the modular computers.

The program is console only, and curator starts with a modular computer with it installed. Requires a lotta paper to print portraits.

- [x] map it
- [x] do the js

## Why It's Good For The Game

Decorate YOUR library!!!!
Put even more of a magnifying glass on the fact that Portraits still cannot be moderated well!!!!
I'm basically admitting now that in the future i'm going to have to be the one to make a painting database moderation panel!!!

## Changelog
:cl:
add: Curators now start with a modular console with a preinstalled app for printing canvases
/:cl:

### CODE REVIEW

File: tgui/packages/tgui/interfaces/NtosPortraitPrinter.js [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+      asset_prefix: "library_private",
+      list: library_private,
+    },
+  ];
+  const tab2list = TABS[tabIndex].list;
+  const current_portrait_title = tab2list[listIndex]["title"];
+  const current_portrait_asset_name = TABS[tabIndex].asset_prefix + "_" + tab2list[listIndex]["md5"];
+  return (
+    <NtosWindow
+      theme="ntos"
```
Comment (stylemistake) [inline tgui/packages/tgui/interfaces/NtosPortraitPrinter.js:37]: looks redundant

**Review by stylemistake — APPROVED** [review]
Isn't this supposed to accompany removal of old shitcode?

**Comment by tralezab** [comment]
It's an entirely new thing, and on purpose too! You should read the lib_machines.dm file and see why I really don't want to touch it.

**Comment by AnturK** [comment]
Should let you print posters with these icons instead. Canvases for priceless originals:tm:

**Comment by carshalash** [comment]
based

**Pull request merged**